action.yml

```diff
@@ -30,15 +34,19 @@ runs:
         git config --global user.name "$(git log -1 --pretty=format:'%an')"
         git remote set-url origin "https://x-access-token:${{ github.token }}@github.com/$GITHUB_REPOSITORY"
       shell: bash
+      working-directory: ${{ inputs.working-directory }}
     - name: Configure trusted publishing credentials
       if: ${{ inputs.setup-trusted-publisher == 'true' }}
       uses: rubygems/configure-rubygems-credentials@v1.0.0
+      working-directory: ${{ inputs.working-directory }}
```

Comment:
This line is extraneous

Reply:
Thanks so much for updating this PR @hsbt wanna address these comments too and help us get this merged? If not, I'm happy to do it and make a new PR.